src/components/Person/Person.jsx

```diff
@@ -1 +1,31 @@
-// export const Person = ({ person }) => ();
+import './Person.scss';
+
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">My name is {name}</h2>
+      {age && <p className="Person__age">I am {age}</p>}
+      <p className="Person__partner">
+        {(() => {
+          if (isMarried) {
+            if (sex === 'm') {
+              return `${partnerName} is my wife`;
+            }
+
+            if (sex === 'f') {
+              return `${partnerName} is my husband`;
+            }
+          }
+
+          return 'I am not married';
+        })()}
```

Comment:
```suggestion
        {isMarried
          ? getMarriedMessage()
          : 'l am not married'}}
```
Consider moving this function from jsx